config/templates/stack-template.json

```diff
@@ -17,19 +17,19 @@
           "{{$subnet.AvailabilityZone}}"
           {{end}}
         ],
-        "DesiredCapacity": "{{.WorkerCount}}",
+        "DesiredCapacity": "{{.WorkerASG.MinSize}}",
```

Comment:
Quick thought but do you prefer entirely removing `DesiredCapacity` from stack-template.json and later improve `kube-aws update` command to support updating DesiredCapacity e.g. `kube-aws update --num-workers 10`?